README.md

```diff
@@ -120,6 +129,7 @@ Done! You can change not only the landing page but any pages like this way. (You
   * `templates` - HTML files used as ejs template. `publish` script converts a markdown file to HTML based on templates under this directory.
 * `dist` - Files compiled by `build` script. `deploy` script deploys a website to GitHub pages based on this directory. Do not change the files under this directory directly.
 * `server` - Files compiled by `build` script. `start` script opens local server based on this directory. Do not change the files under this directory directly.
+* `doc-translation` - The folder contains documentation in several languages written in markdown format.
```

Comment:
What do you think about placing `README-IND.md` at the root level?

Reply:
> What do you think about placing `README-IND.md` at the root level?

I don't know, what do you think? :"

Reply:
I think it would be better for accessibility. Can you move the `README-IND.md` file to the root?

Reply:
> I think it would be better for accessibility. Can you move the `README-IND.md` file to the root?

great idea, of course I'll do it

Reply:
@parksb I just moved the README-IND.md file to root, is this correct? What do you think? :"

I also changed the link in the README.md file (the link that will display the README-IND.md file)

Reply:
By the way, sorry if there is a mistake
I'm not very experienced in contributing to open source projects yet😅

Reply:
That's good 🙂 I left two comments suggesting little changes.

Reply:
That's good 🙂 I left two comments suggesting little changes.